tfe/testing.go

```diff
@@ -241,3 +243,13 @@ func retry(maxRetries, secondsBetween int, f retryableFn) (interface{}, error) {
 		retries += 1
 	}
 }
+
+// Allows skipping the given TestCheckFunc unless beta checks enabled.
+func betaOnlyCheck(testFunc resource.TestCheckFunc) resource.TestCheckFunc {
+	if betaFeaturesEnabled() {
+		return testFunc
+	}
+	return func(state *terraform.State) error {
+		return nil
+	}
+}
```

Comment:
Love this :fire:

Reply:
Thanks, I've been quite enjoying the testing framework!